css/mistica-common.css

```diff
@@ -695,3 +705,90 @@ button.mistica-display-card:has(.mistica-card__media):active:after {
 .mistica-card .mistica-card__buttons {
     flex: 1;
 }
+
+/* Accordion */
+
+.mistica-accordion-item {
+    interpolate-size: allow-keywords;
+}
+
+.mistica-accordion-item:after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 1px;
+    background-color: var(--vcolor-divider);
+}
+
+.mistica-accordion-item:last-child:after {
+    display: none;
+}
+
+.mistica-accordion-item__summary {
+    cursor: pointer;
+    min-height: 72px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 16px;
+    margin: 0 -16px;
+    transition: background-color 0.1s ease-in-out;
+}
+
+.mistica-accordion-item__summary-body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 2px;
+}
+
+.mistica-accordion-item__summary-title {
+    font-size: var(--mistica-font-size-3);
+    line-height: var(--mistica-line-height-3);
+    font-weight: var(--mistica-font-weight-3);
+    color: var(--vcolor-textPrimary);
+}
+
+.mistica-accordion-item__summary-subtitle {
+    font-size: var(--mistica-font-size-2);
+    line-height: var(--mistica-line-height-2);
+    font-weight: var(--mistica-font-weight-2);
+    color: var(--vcolor-textPrimary);
+}
+
+.mistica-accordion-item__chevron {
```

Comment:
Size decreases with long title / subtitle

<img width="908" alt="Screenshot 2025-04-11 at 19 04 34" src="https://github.com/user-attachments/assets/75bfec7f-2da7-4b51-a6a9-2817297d0a86" />

Reply:
Good catch, there was a missing flex-shrink: 0
Should be fixed now